plugins/pulp_rpm/plugins/importers/yum/existing.py

```diff
@@ -47,6 +47,7 @@ def check_repo(wanted):
                                       ids.TYPE_ID_DRPM)
 
         unit_generator = (model(**unit_tuple._asdict()) for unit_tuple in values)
+        # FIXME this function being called doesn't have a fields parameter
```

Comment:
Is this still a FIXME or did it get it get fixed with the recent platform PRs you made?